scripts/st2bootstrap-deb.sh

```diff
@@ -339,17 +339,17 @@ configure_st2_user () {
     sudo useradd stanley
   fi
 
-  sudo mkdir -p /home/stanley/.ssh
+  sudo mkdir -p ~stanley/.ssh
```

Comment:
Yeah, I agree.

In theory I'm not against this change, but it's a bit more involved and we need to make sure it doesn't break anything.

Also, IIRC, `~` and `$HOME` usually doesn't contain trailing slash by default so if anything it should be `~/stanley` to be on the safe side.